libp2p/transports/wstransport.nim

```diff
@@ -132,6 +134,16 @@ method start*(
   await procCall Transport(self).start(addrs)
   trace "Starting WS transport"
 
+  # use autotls if possible (and if cert/key not specified)
+  if not self.secure and not self.autotls.isNil():
+    try:
+      self.tlsCertificate = await self.autotls.getCertWhenReady()
+      self.tlsPrivateKey = self.autotls.getTLSPrivkey()
+    except AutoTLSError as exc:
+      raise newException(LPError, exc.msg, exc)
+    except TLSStreamProtocolError as exc:
+      raise newException(LPError, exc.msg, exc)
```

Comment:
it would be better to do this if `isWss == true`

Reply:
`isWss` is set afterwards and for it to be `true` requires `secure()` to return true, which in turn requires the certificate and key to be set already.

Reply:
it also requires `/wss` address right?